Components/BlockPostFooter/_style.scss

```diff
@@ -23,6 +23,7 @@
 
     &-info {
       display: flex;
+      flex: 1;
```

Comment:
should maybe be explicitly `flex: 1 1 auto;` to prevent any IE11 issues with the third parameter for flex-basis

Reply:
Thanks, I'm not familiar with this, but I added `flex: 1` to specifically fix a bug with IE11 where the author text wasn't wrapping, but overflowing the outer container.

Reply:
Not sure if it applies here, but here is an example of what I mean: https://makandracards.com/makandra/54644-do-not-use-flex-1-or-flex-basis-0-inside-flex-direction-column-when-you-need-to-support-ie11

Reply:
I tried the suggested fixes. However, they only end up breaking it in IE11 again. Any other suggestions? Otherwise, I would keep it the way it is.